ansible/roles/foundations/aws_cd/files/code_pipeline_ps_hippo.yml

```diff
@@ -101,14 +101,14 @@ Resources:
       Artifacts:
         Location: !Sub "artefacts.${HostedDomainPublic}"
         Type: S3
-        Name: ps_hippo.zip
+        Name: hippo.zip
         NamespaceType: NONE
-        Path: ps-hippo
+        Path: hippo
         Packaging: ZIP
-      Description: Package ps-hippo repo for other builds
+      Description: Package hippo repo for other builds
       Environment:
         ComputeType: BUILD_GENERAL1_SMALL
-        Image: woledzki/alpine-virtualenv:latest
+        Image: selenium/standalone-chrome:latest
```

Comment:
What's the implication of this? This seems unrelated to the other changes. Should it be in this PR?